front/src/applications/stdcm/components/SimulationReportSheet.tsx

```diff
@@ -0,0 +1,353 @@
+            <View style={styles.rcInfo.stdcmApplication}>
+              <Text style={styles.rcInfo.applicationDate}>{t('applicationDate')}</Text>
+              <Text style={styles.rcInfo.date}>{formatDay()}</Text>
+              <Text style={styles.rcInfo.referencePath}>{t('referencePath')}</Text>
+              <Text style={styles.rcInfo.pathNumber}>{fakeInformation.path_number1}</Text>
+            </View>
+          </View>
+        </View>
+        <View style={styles.convoyAndRoute.convoyAndRoute}>
+          <View style={styles.convoyAndRoute.convoy}>
+            <Text style={styles.convoyAndRoute.convoyTitle}> {t('convoy')}</Text>
+            <View style={styles.convoyAndRoute.convoyInfo}>
+              <View style={styles.convoyAndRoute.convoyInfoBox1}>
+                <Text style={styles.convoyAndRoute.convoyInfoTitles}>{t('speedLimitByTag')}</Text>
+                <Text style={styles.convoyAndRoute.convoyInfoData}>
+                  {stdcmData.simulation.speed_limit_tags || '-'}
+                </Text>
+                <Text style={styles.convoyAndRoute.convoyInfoTitles}>{t('towedMaterial')}</Text>
+                <Text style={styles.convoyAndRoute.convoyInfoData}>-</Text>
+                <Text style={styles.convoyAndRoute.convoyInfoTitles}>{t('maxSpeed')}</Text>
+                <Text style={styles.convoyAndRoute.convoyInfoData}>
+                  {`${rollingStockData.max_speed * 3.6} km/h`}
+                </Text>
+              </View>
+              <View style={styles.convoyAndRoute.convoyInfoBox2}>
+                <Text style={styles.convoyAndRoute.convoyInfoTitles}>{t('maxWeight')}</Text>
+                <Text style={styles.convoyAndRoute.convoyInfoData}>
+                  {`${Math.floor(rollingStockData.mass / 1000)} t`}
+                </Text>
+                <Text style={styles.convoyAndRoute.convoyInfoTitles}>{t('referenceEngine')}</Text>
+                <Text style={styles.convoyAndRoute.convoyInfoData}>
+                  {rollingStockData.metadata?.reference || '-'}
+                </Text>
+                <Text style={styles.convoyAndRoute.convoyInfoTitles}>{t('maxLength')}</Text>
+                <Text
+                  style={styles.convoyAndRoute.convoyInfoData}
+                >{`${rollingStockData.length} m`}</Text>
+              </View>
+            </View>
+          </View>
+          <View style={styles.convoyAndRoute.route}>
+            <Text style={styles.convoyAndRoute.routeTitle}>{t('requestedRoute')}</Text>
+            <View style={styles.convoyAndRoute.fromBanner}>
+              <View style={styles.convoyAndRoute.fromBox}>
+                <Text style={styles.convoyAndRoute.from}>{t('from')}</Text>
+              </View>
+              <Text style={styles.convoyAndRoute.fromNumber}>{fakeInformation.path_number1}</Text>
+              <Text style={styles.convoyAndRoute.fromScheduled}>
+                {t('scheduledArrival')} {arrivalTime}
```

Comment:
It's better to include the variable in the translation. It's the same for depatureTime, l206.
```suggestion
                {t('scheduledArrival', arrivalTime)}
```

Reply:
done